### PR TITLE
updating for Try .NET -- final set of first round

### PR DESCRIPTION
**Pull request by WilliamAntonRohm**
Docs PR is https://github.com/dotnet/dotnet-api-docs/pull/3894

### CODE REVIEW

**Comment by WilliamAntonRohm** [comment]
(not ready for review yet)

**Comment by WilliamAntonRohm** [comment]
#hold-off

**Comment by WilliamAntonRohm** [comment]
@mairaw, @BillWagner -- Maira or Bill, please review these changes -- this should be the last set of samples for this first round of Try .NET.

I'll have the doc PR ready tomorrow.

File: snippets/csharp/VS_Snippets_CLR_Classic/classic DateTime.ToString2 Example/CS/source.cs [hunk 1]
```diff
@@ -1,9 +1,11 @@
-﻿// <Snippet1>
+﻿        // <Snippet1>
```
Comment (mairaw) [inline snippets/csharp/VS_Snippets_CLR_Classic/classic DateTime.ToString2 Example/CS/source.cs]: This now has two snippet1 tags

File: snippets/csharp/VS_Snippets_CLR_System/System.Globalization.CultureInfo/cs/Change1.cs [hunk 2]
```diff
@@ -1,11 +1,11 @@
-﻿// <Snippet3>
-using System;
+﻿using System;
```
Comment (mairaw) [inline snippets/csharp/VS_Snippets_CLR_System/System.Globalization.CultureInfo/cs/Change1.cs]: we could try to use the new syntax with this one and see if that would work. I'll make a change on the PR for us to test that.

**Comment by mairaw** [comment]
I'm going to close this PR for now. We should try to use the new syntax, which wouldn't require sample changes.

File: snippets/csharp/VS_Snippets_CLR_Classic/classic DateTime.ToString2 Example/CS/source.cs (hunk 1 above)
Comment (WilliamAntonRohm) [reply snippets/csharp/VS_Snippets_CLR_Classic/classic DateTime.ToString2 Example/CS/source.cs]: Fixed.

**Pull request closed without merging**